src/main/java/com/google/cloud/tools/gradle/endpoints/framework/server/EndpointsServerPlugin.java

```diff
@@ -33,14 +34,15 @@
  * Plugin definition for Endpoints Servers (on App Engine) for generation of
  * client libraries and discovery docs.
  *
- * Also provides the artifact "discovery-docs" that is a zip of all the discovery
- * docs that this server exposes (as defined in web.xml)
+ * Also provides the artifact "{@value ARTIFACT_CONFIGURATION}" that is a zip
```

Comment:
yeah that might be the correct way to do it, intellij seemed okay with no #, I updated it anyway